tap_closeio/context.py

```diff
@@ -46,7 +46,10 @@ def set_bookmark(self, path, val):
 
     def get_offset(self, path):
         off = bks_.get_offset(self.state, path[0])
-        return (off or {}).get(path[1])
+        value = (off or {}).get(path[1])
+        if path[0] == "activities" and value > 250000:
```

Comment:
Add a comment here regarding a specific scenario like when the state reached this condition for the customer with the previous version and this will reduce skip to 0 for date_windowing and all.

Reply:
Added comment